docs/development/local_setup.md

```diff
@@ -310,34 +314,53 @@ to operate against your local running Gardener API Server.
-- It was not possible to create two or more Shoot clusters in parallel.
-- The communication between the Seed and the Shoot Clusters uses VPN tunnel. In this setup tunnels are not needed since all components run on localhost.
+The next steps require modifications of the example manifests. These modifications are part of local setup and should not be `git push`-ed. To do not interfere with git, let's copy the example manifests to `dev/` which is ignored by git.
 
-## Additional information
+```bash
+cp example/*.yaml dev/
+```
 
-To make sure that a specific Seed cluster will be chosen, specify the `.spec.seedName` field (see [here](../../example/90-shoot.yaml#L265-L266) for an example Shoot manifest).
+#### 2. Create a Project
 
-Please take a look at the [example manifests folder](../../example) to see which resource objects you need to install into your Garden cluster.
+Every Shoot is associated with a Project. Check the corresponding example manifests `dev/00-namespace-garden-dev.yaml` and `dev/05-project-dev.yaml`. Adapt them and create them.
 
+```bash
+kubectl apply -f dev/00-namespace-garden-dev.yaml
+kubectl apply -f dev/05-project-dev.yaml
+```
+
+Make sure that the Project is successfully reconciled:
+
+```bash
+$ k get project dev
+NAME   NAMESPACE    STATUS   OWNER                  CREATOR            AGE
+dev    garden-dev   Ready    john.doe@example.com   kubernetes-admin   6s
+```
 
-# Appendix
+#### 3. Create a CloudProfile
+
+The `CloudProfile` resource is provider specific and describes the underlying cloud provider (available machine types, regions, machine images, etc.). Check the corresponding example manifest `dev/30-cloudprofile.yaml`. Check also the documentation and example manifests of the provider extension. Adapt `dev/30-cloudprofile.yaml` and apply it. 
+
+```bash
+kubectl apply -f dev/30-cloudprofile.yaml
+```
 
-## Configure Seed clusters for local development
+#### 4. Create the required ControllerRegistrations
+
+[Known Extension Implementations](../../README.md#infrastructure-provider) contains a list of available extension implementations. You need to create a ControllerRegistration for at least one infrastructure provider, dns provider (if the DNS for the Seed is not disabled), at least one operating system extension and at least one network plugin extension.
+As a convention, example ControllerRegistration manifest for an extension is located under `example/controller-registration.yaml` in the corresponding repository (for example for AWS the ControllerRegistration can be found [here](https://github.com/gardener/gardener-extension-provider-aws/blob/master/example/controller-registration.yaml)). An example creation of ControllerRegistration for provider-aws:
+
+```bash
+kubectl apply -f https://raw.githubusercontent.com/gardener/gardener-extension-provider-aws/$VERSION/example/controller-registration.yaml
+```
+
+#### 5. Create a Seed
```

Comment:
Should we call this "Register a Seed" to make more clear that Gardener does not create the seeds itself?